inc/rulematchedlog.class.php

```diff
@@ -300,6 +306,22 @@ function showForm($items_id, $itemtype) {
             }
          }
          echo "</td>";
+         echo "<td>";
+         $criteria = $dbu->importArrayFromDB($data['criteria']);
+         if (count($criteria) > 0) {
+            echo "<ul style='list-style-type:disc'>";
+            foreach ($criteria as $key=>$value) {
+               echo "<li>".$key.": ";
+               if (is_array($value)) {
+                  echo implode("<br>", $value);
+               } else {
+                  echo $value;
+               }
+               echo "</li>";
+            }
+            echo "</ul><hr style='border: 0;height: 1px;background-image: -webkit-linear-gradient(left, #f0f0f0, #8c8b8b, #f0f0f0);background-image: -moz-linear-gradient(left, #f0f0f0, #8c8b8b, #f0f0f0);background-image: -ms-linear-gradient(left, #f0f0f0, #8c8b8b, #f0f0f0);background-image: -o-linear-gradient(left, #f0f0f0, #8c8b8b, #f0f0f0);'>";
```

Comment:
Putting this in a CSS file would be better.

Reply:
Transfered in the css